lms/config.py

```diff
@@ -116,6 +116,9 @@ def __post_init__(self):
     _Setting("mailchimp_digests_subaccount"),
     _Setting("mailchimp_digests_email"),
     _Setting("mailchimp_digests_name"),
+    _Setting("mailchimp_annotation_activity_subaccount"),
```

Comment:
New subaccount for these emails. Instead of scoping the name to "mentions" it's a bit more general, using "annotation activity".